dataworkspace/dataworkspace/apps/core/views.py

```diff
@@ -53,6 +53,8 @@ class SupportView(FormView):
     form_class = SupportForm
     template_name = 'core/support.html'
 
+    ZENDESK_TAGS = {'data-request': "data_request"}
```

Comment:
Different quotes on a single line is a new one ;)
```suggestion
    ZENDESK_TAGS = {"data-request": "data_request"}
```

Reply:
😓 maybe we should enable `black` to enforce double quotes

Reply:
Fine by me